core/src/main/java/org/apache/struts2/views/jsp/SetTag.java

```diff
@@ -59,4 +61,32 @@ public void setScope(String scope) {
     public void setValue(String value) {
         this.value = value;
     }
+
+    public void setDoNotTrimBody(String doNotTrimBody) {
+        this.doNotTrimBody = Boolean.parseBoolean(doNotTrimBody);
+    }
+
+    public void setStripBodyLineBreaks(String stripBodyLineBreaks) {
+        this.stripBodyLineBreaks = Boolean.parseBoolean(stripBodyLineBreaks);
+    }
+
+    @Override
+    protected String getBody() {
+        if (bodyContent == null) {
```

Comment:
This can be probably rewritten to use `super.getBody()` instead of duplicating code here.

Reply:
This has been addressed in the most recent commit (the `super.getBody()` is now called in `getBody()`, along with improved logic to ensure behaviour for nulls is the same as the original tag).